djTaskLine/dj_app/dj_main/settings/base.py

```diff
@@ -36,6 +36,8 @@
 ACCOUNT_EMAIL_REQUIRED = True
 ACCOUNT_AUTHENTICATION_METHOD = 'email'
 ACCOUNT_USERNAME_REQUIRED = False
+# ACCOUNT_ADAPTER = 'apps.users.adapter.CustomAccountAdapter'
+# LOGIN_REDIRECT_URL = '/'
```

Comment:
Made default redirect on '/'. Still have problem with detecting user auth status on front.
Wanted to extend LoginView to put some additional cookies to let frontend know that user has logged in